python/tvm/dlight/gpu/default_schedule.py

```diff
@@ -0,0 +1,89 @@
+# pylint: disable=missing-docstring
+from typing import Callable
+
+from tvm import tir
+from tvm._ffi import get_global_func
+from tvm.ir import IRModule
+from tvm.ir.transform import PassContext, module_pass
+from tvm.target import Target
+
+
+def _default_schedule(  # pylint: disable=too-many-locals
+    func: tir.PrimFunc,
+    target: Target,
+) -> tir.PrimFunc:
+    max_threads_per_block = None
+    for name in ["max_threads_per_block", "max_num_threads"]:
+        if max_threads_per_block is None:
+            max_threads_per_block = target.attrs.get(name, None)
+    if max_threads_per_block is None:
```

Comment:
Should have one schedule_rule, and invoke that schedule rule here. The schedule do not need to match all instances. Dlight is meant to get performant schedules

Reply:
that makes sense

Reply:
i'm going to remove this rule first. seems not super relevant to this PR